Python/codegen.c

```diff
@@ -2884,6 +2938,14 @@ codegen_stmt_expr(compiler *c, location loc, expr_ty value)
     return SUCCESS;
 }
 
+#define CODEGEN_COND_BLOCK(FUNC, C, S) \
+    do { \
+        _PyCompile_EnterConditionalBlock((C)); \
+        int __result = FUNC((C), (S)); \
```

Comment:
Since you're in a do-while block, redefinition here doesn't really matter and __NAME variables are (whether lower or upper case) reserved by the standards. So just "int rc = ..." should be sufficient.

(I think the compiler would be satisfied when we use that macro in the case body even if the case body isn't enclosed by more {})